DictDataGenerator.py

```diff
@@ -0,0 +1,92 @@
+import random
+import datetime
+import argparse
+import csv
+import os
+from DataGenerator import DataGenerator
+from DictDataClasses import DictDataClasses
+
+class DictDataGenerator(DataGenerator):
+
+    def __init__(self):
+        # Generate random inst ids with the prefix of the bank (ABC) and a random string composed of numbers and letters
+        self.dict = DictDataClasses().get_dict()
+
+    # In DataConfiguration.py, 'Data Args' field should look like:
+    # {'Type': 'position'}
+    def run(self, args):
+        type = args["Type"]
+        if type == 'price':
+            return self.__generate_price_entity()
+        elif type == 'position':
+            return self.__generate_position_entity()
+        elif type == 'inst-ref':
+            return self.__generate_inst_ref_entity()
+
+    @staticmethod
+    def main():
+        args = DictDataGenerator.__get_args()
```

Comment:
This line seems wrong. Why call a class like this? Is there another way you could implement this? It likely seems that the main function should not exist within the class itself?

Reply:
`__get_args` is a statics method - should I have it in a Util.py file?